cli/common_test.go

```diff
@@ -63,6 +63,29 @@ func Test_Common_createRequest(t *testing.T) {
 				},
 			},
 		},
+
+		// This test ensures that loading a single unit from a directory with the
+		// slice expression "@1", "@foo" and "@5" results in the expected
+		// controller request.
+		{
+			Setup: []testFileSystemSetup{
```

Comment:
FYI: we could return this standard test setup into a function which we just repeat here:

```
{ 
  Setup: GivenSimpleSingleUnitFileSystemSetup(),
  [..]
```